app/controllers/account/tasks_controller.rb

```diff
@@ -105,7 +105,7 @@ def remove_attachment
   def complete
     @project = parent
     @task = resource
-    @task.update(complete: true)
+    @task.update(complete: true, completed_at: Time.now)
```

Comment:
you don't need complete field, if a task has completed_at that means it completed

Reply:
completed = completed_at not nil